plugins/pulp_rpm/plugins/distributors/export_distributor/export_utils.py

```diff
@@ -109,6 +109,10 @@ def validate_export_config(config):
             if not os.path.isabs(value):
                 msg = _("Value for 'export_dir' must be an absolute path: %s" % value)
                 return False, msg
+        if key == constants.RELATIVE_URL_KEYWORD:
```

Comment:
this will work only for API, you forgot to add the option for pupulp-admin https://github.com/seandst/pulp_rpm/blob/1394-relative-url-export-config/extensions_admin/pulp_rpm/extensions/admin/export.py#L53